src/fides/api/main.py

```diff
@@ -22,14 +22,7 @@
 
 from fides.api.ctl import view
 from fides.api.ctl.database.database import configure_db
-from fides.api.ctl.routes import (
-    admin,
-    crud,
-    datamap,
-    generate,
-    health,
-    validate,
-)
+from fides.api.ctl.routes import admin, crud, datamap, generate, health, validate
```

Comment:
Is there something we can set to preserve these import lists? Not sure what the rest of the team thinks, I find it very useful to see diffs per-line in cases like this with imports and method args

Reply:
this is auto-formatting, maybe there's a setting we could dig into but that should be its own issue